apis/core/v1alpha1/flagsourceconfiguration_types.go

```diff
@@ -47,7 +47,7 @@ const (
 	defaultSocketPath                string           = ""
 	defaultEvaluator                 string           = "json"
 	defaultImage                     string           = "ghcr.io/open-feature/flagd"
-	defaultTag                       string           = "v0.4.5"
```

Comment:
can we remove this from this PR and do it in a different one? otherwise this tag bump will not be visible in the changelog